lib/config/context.js

```diff
@@ -69,12 +69,8 @@ class Context {
     if (user.token) {
       api.headers.Authorization = `Bearer ${user.token}`;
     }
-    if (user.auth_provider && user.auth_provider.token) {
-      const auth_provider = user.auth_provider;
-      api.headers.Authorization = `Bearer ${auth_provider.token}`;
```

Comment:
I wonder if we're not loosing the support for bare auth provider token?

Reply:
I think we should be good on that part. Just tested and it worked. I think I was originally trying to validate the kube config which I don't think Kubebox should do so I removed that logic.